scripts/modules/elastic_stack.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, **options):
                             ("apm-server.kibana.{}".format(cfg), default_apm_server_creds.get(cfg)))
 
         if self.options.get("enable_kibana", True):
-            self.depends_on["kibana"] = {"condition": "service_healthy"}
```

Comment:
ditto

Reply:
Reverted this change. Resolving comment thread.